draft-ietf-webtrans-http3.md

```diff
@@ -432,6 +432,10 @@ implement a fairness scheme that ensures that each transport within connection
 gets a reasonable share of controlled resources; this applies both to sending
 data and to opening new streams.
 
+A client may attempt to exhaust resources by opening too many WebTransport
+sessions at once.  In cases when the client is untrusted, the user agent SHOULD
```

Comment:
I don't understand this, who is the client if there is a user-agent in the picture?

Reply:
https://www.ietf.org/archive/id/draft-ietf-webtrans-overview-04.html#name-conventions-and-definitions-2

Reply:
Ah, thanks.